src/seedu/addressbook/AddressBook.java

```diff
@@ -1146,12 +1124,12 @@ private static String getUsageInfoForExitCommand() {
     /**
      * Removes sign(p/, d/, etc) from parameter string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
+     * @param fullString  Parameter as a string
+     * @param prefix  Parameter sign to be removed
```

Comment:
Good idea to change the comments. Note that you should probably remove the mention of "sign" here too.